libs/gstsavantframemeta/pygstsavantframemeta/__init__.py

```diff
@@ -173,3 +173,11 @@ def add_pad_probe_to_unpack_and_move_batch(
         video_pipeline.memory_handle,
         stage,
     )
+
+
+def add_pad_probe_to_remove_tracker_objs(element: Gst.Element):
```

Comment:
Why this method accepts `Gst.Element` and not `Gst.Pad` like the other `add_pad_probe_` methods?

Reply:
I see no reason leaving `gst_element_get_static_pad` call on the python side.

Reply:
Is the comment about the naming?

Reply:
> I see no reason leaving `gst_element_get_static_pad` call on the python side.

* The method `add_pad_probe_to_remove_tracker_objs` basically a wrapper for `gst_pad_add_probe`, so it makes sense to pass only pad, not the whole element.
* It would be consistent with the other methods to pass only a pad. For any method to add a pad probe we are calling `gst_element_get_static_pad` on the python side.

> Is the comment about the naming?
No, the name is ok.

Reply:
Refactored the bindings to pass a pointer to pad from the python side.